src/blueapi/cli/cli.py

```diff
@@ -15,16 +15,15 @@
 
 from blueapi import __version__
 from blueapi.cli.format import OutputFormat
+from blueapi.cli.scratch import setup_scratch
+from blueapi.cli.updates import CliEventRenderer
```

Comment:
Could: If we're against relative imports as a policy, is there any way we can make the linting fail if they are present?